app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -0,0 +1,194 @@
+package com.github.wnder;
+
+import android.animation.ObjectAnimator;
+import android.animation.TypeEvaluator;
+import android.animation.ValueAnimator;
+import android.content.Intent;
+import android.graphics.BitmapFactory;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.mapbox.geojson.Feature;
+import com.mapbox.geojson.Point;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.maps.Style;
+
+import com.mapbox.mapboxsdk.style.layers.PropertyFactory;
+import com.mapbox.mapboxsdk.style.sources.GeoJsonSource;
+
+import com.mapbox.mapboxsdk.style.layers.SymbolLayer;
+
+
+public class GuessLocationActivity extends AppCompatActivity implements OnMapReadyCallback, View.OnClickListener, MapboxMap.OnMapClickListener{
+
+    private MapView mapView;
+    private MapboxMap mapboxMap;
+    private LatLng currentPosition = new LatLng(64.900932, -18.167040);

```

Comment:
We will need a Location object at some point (as this is what we wanted to use instead of google or mapbox LatLng). But I guess we can convert once we leave this activity.